funnel/views/membership.py

```diff
@@ -540,7 +540,7 @@ def edit(self):
                 return (
                     {
                         'status': 'error',
-                        'message': _("At lease one role must be chosen"),
+                        'error_description': _("At lease one role must be chosen"),
```

Comment:
Apart from the typo ("lease"), switch to active voice: "Please pick one or more roles"